### PR TITLE
Branch 165340781

### PR DESCRIPTION
**Pull request by rmlarsen**

### CODE REVIEW

**Comment by rmlarsen** [comment]
@tensorflower-gardener test this please

**Review by gunan — APPROVED** [review]

File: tensorflow/contrib/learn/python/learn/estimators/estimator.py [hunk 1]
```diff
@@ -337,20 +344,27 @@ def _write_dict_to_summary(output_dir,
...
+    elif isinstance(dictionary[key], six.string_types):
+      try:
+        summ = summary_pb2.Summary.FromString(dictionary[key])
+        for i, _ in enumerate(summ.value):
+          summ.value[i].tag = key
+        summary_proto.value.extend(summ.value)
+      except message.DecodeError:
+        logging.warn('Skipping summary for %s, cannot parse string to Summary.',
+                     key)
+        continue
```
Comment (terrytangyuan) [inline tensorflow/contrib/learn/python/learn/estimators/estimator.py]: @martinwicke How are you planning to sync changes from contrib and to core? (trying to track all the changes related to estimators)

**Comment by martinwicke** [comment]
We will deprecate contrib and eventually delete it. I don't plan on syncing
changes.

**Comment by terrytangyuan** [comment]
Sounds of. Was just wondering since still there seems to be some commits to contrib.

**Comment by martinwicke** [comment]
Yes. We're quite close to having everything complete in core. The last
missing piece is Experiment. Then the deprecations will start.

**Pull request merged**